src/applications/vre/28-1900/tests/e2e/chapter31-maximal.cypress.spec.js

```diff
@@ -36,12 +37,15 @@ const testConfig = createTestConfig(
         cy.get('.usa-button-primary').click();
         cy.get('.usa-button-primary').click();
         cy.get('.usa-button-primary').click();
+        cy.get('.usa-button-primary').click();
+        cy.get('.usa-button-primary').click();
+        cy.get('.usa-button-primary').click();
```

Comment:
So many `.usa-button-primary` clicks! For future readability, it might be better to use `cy.findByText` to target the named button... or do they all have the same name too?